docs/outputs/output_files.md

```diff
@@ -0,0 +1,130 @@
+# Output Files
+
+After running the workflow, the `/path/to/output/dir` folder will contain a 
+`logs` folder for troubleshooting, as well as additional folders associated with
+the tools that were used to generate files (e.g. `freesurfer`, `labelmerge`),
+but for most purposes, all the primary outputs of interest will be in the 
+`mrtrix` directory with the following structure:
+
+```
+mrtrix/
+└── sub-{subject}
+    ├── dti
+    ├── dwi
+    ├── response
+    └── tractography
+```
+
+Briefly, `dti` containers processed diffusion tensor images along with 
+quantitative maps (e.g. `FA`, `MD`, etc.), `dwi` contains images that were
+converted from Nifti (`.nii.gz`) to the MRtrix imaging format (`.mif`), 
+`response`, contains subject-specific response functions, and `tractography`
```

Comment:
`response`, -> `response` (drop the comma)

Reply:
Fixed in eed5224